graphics/font.cpp

```diff
@@ -466,4 +466,15 @@ Common::String Font::handleEllipsis(const Common::String &input, int w) const {
 	}
 }
 
+TextAlign convertTextAlignH(TextAlign alignH, bool rtl) {
+	switch (alignH) {
+		case kTextAlignStart:
+			return rtl ? kTextAlignRight : kTextAlignLeft;
+		case kTextAlignEnd:
+			return rtl ? kTextAlignLeft : kTextAlignRight;
+		default:
+			return alignH;
+	}
+}
```

Comment:
This is the center of abstraction.
Everywhere alignment is fetched during creation, now we can also use `kTextAlignStart` & `kTextAlignEnd`. When adding a widget, `kTextAlignStart` is used by default instead of `kTextAlignLeft` 

When RTL is turned off:
Start - means LeftAlign
End - means RightAlign

When RTL is turned on:
Start - means RightAlign
End - means LeftAlign

What this effectively does is centers all our alignment code, so we don't need to have bits and pieces of manual alignment settings.
(manual alignment can override these if needed). the default switch case returns the original value, so manual alignment still works.